src/lib/controllers/ui-controller.js

```diff
@@ -762,6 +776,7 @@ export default class UIController {
       showPanelTab: this.showPanelTab.bind(this),
       togglePanelTab: this.togglePanelTab.bind(this),
       registerAndActivateGetSelectedText: this.registerAndActivateGetSelectedText.bind(this),
+      registerAndActivateMouseMove: this.registerAndActivateMouseMove.bind(this),
```

Comment:
Separated it from the main RegisterEvent, because MouseMove should be activated after settings are loaded